CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 2.8)
+cmake_minimum_required (VERSION 2.8.12)
```

Comment:
It's also possible to leave the minimum at 2.8; it just means we'd have to drop `INTERFACE` from the `target_link_libraries` line. Would you prefer that?